docs/content/preview/troubleshoot/cluster/failed_tablets.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Recover YB-TServer from crash loop
+linkTitle: Recover YB-TServer from crash loop
+description: Recover YB-TServer from crash loop
+menu:
+  preview:
+    parent: troubleshoot-cluster
+    weight: 866
+isTocNested: true
+showAsideToc: true
+---
+
+When a YB-TServer process or node has failed, YugabyteDB automatically triggers a remote bootstrap for
+ most types of tablet data corruption or failures. However, there is still a number of cases a number of cases where this automatic mechanism might not be able to help.
```

Comment:
there is still a number of cases a number of cases where  >

there is still a number of cases where